applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh

```diff
@@ -4,8 +4,12 @@ cmake .. \
 -DSCOTCH_INCLUDE_DIR="/usr/include/scotch/"	            \
 -DSCOTCH_LIBRARY="libscotchmetis-5.1.so"	            \
 -DSCOTCHERR_LIBRARY="libscotcherr-5.1.so"	            \
--DCMAKE_CXX_FLAGS="-O3 -mavx2 -fPIC -fopenmp"               \
--DCMAKE_C_FLAGS="-O3 -mavx2 -fPIC -fopenmp"                 \
+-DCMAKE_CXX_FLAGS="-O3 -fPIC -fopenmp"               \
+-DCMAKE_C_FLAGS="-O3 -fPIC -fopenmp"                 \
```

Comment:
just to make it clear, we should have something that is very compatible, so i would not push any flag which refers to a specific set of instructions